linux_os/guide/services/ssh/ssh_server/disable_host_auth/ansible/shared.yml

```diff
@@ -3,9 +3,4 @@
 # strategy = restrict
 # complexity = low
 # disruption = low
-- name: Disable Host-Based Authentication
-  lineinfile:
-    create: yes
-    dest: /etc/ssh/sshd_config
-    regexp: ^HostbasedAuthentication
-    line: HostbasedAuthentication no
+{{{ ansible_sshd_set('', "HostbasedAuthentication", "no") }}}
```

Comment:
It would be better to make  `msg` an optional argument because you always pass an empty string there anyway.

Reply:
Yeah that's a good change. That and documentation are the two things I want to do still in this patch. 

Other than the two sets of rules I've changed, I don't think I'll track down everywhere these new macros can be used.